### PR TITLE
Switch translation of ruby function calls to allow nested calls

### PR DESCRIPTION
**Pull request by sorig**
Fixes issue with ruby in #4154

### CODE REVIEW

**Comment by sorig** [comment]
@karlnapf just seeing if this passes. May solve the issue you were seeing

**Comment by karlnapf** [comment]
Just merged #4154 if you wanna rebase

**Comment by karlnapf** [comment]
You will need to test that, I removed the nested call in the `kwargs.sg`

**Comment by sorig** [comment]
Just rebased and added the test

File: examples/meta/src/meta_api/kwargs.sg [hunk 1]
```diff
@@ -22,14 +22,11 @@
...
-
-
 
 # Real example
 # ------------
 
 Kernel k = kernel("GaussianKernel", log_width=2.0)
 KernelMachine svm = kernel_machine("LibSVM", C1=1.1, kernel=k)
 
+KernelMachine svm2 = kernel_machine("LibSVM", kernel=kernel("GaussianKernel"))
```
Comment (karlnapf) [inline examples/meta/src/meta_api/kwargs.sg]: can also put the scalars in there :) we have some hackery that covers it now

Comment (karlnapf) [reply examples/meta/src/meta_api/kwargs.sg]: but of course not necessary

**Comment by karlnapf** [comment]
Thanks for the quick fix! :)

**Comment by karlnapf** [comment]
Seems like ruby likes you :)

**Pull request merged**